data/vtlib/final_output_prep.json

```diff
@@ -305,28 +299,22 @@
 		"name":"c2a.json"
 	},
 	{
-		"id":"scramble",
+		"id":"bam2cram",
+		"description":"converts BAM input (autodetected) to CRAM output",
 		"type":"EXEC",
 		"use_STDIN": true,
 		"use_STDOUT": true,
-                "cmd":[
-			"scramble",
-			{"subst":"b2c_mt", "ifnull":{"subst_constructor":{ "vals":[ "-t", {"subst":"b2c_mt_val"} ]}}},
-			{"subst":"b2c_fmtver", "ifnull":{"subst_constructor":{ "vals":[ "-V", {"subst":"b2c_format_version"} ]}}},
-			{"subst":"b2c_compress_level", "ifnull":"-7"},
-			"-I", "bam",
-			"-O", "cram",
-			{"subst":"scramble_reference_flag"},
-			{"select":"scramble_embed_reference","default":0,"select_range":[0,1],
-				"cases":[
-					[],
-					"-e"
-				]
-			}
-		]	
+		"cmd": [
+			{"subst":"samtools_executable"}, "view", "-C",
+			{"subst_constructor":{"vals":[ "--threads", {"subst":"b2c_threads","ifnull":{"subst":"b2c_mt_val"}}]}},
+			{"subst_constructor":{"vals":[ "--output-fmt-option", {"subst_constructor":{"vals":[ "level", {"subst":"b2c_compression_level","ifnull":7} ], "postproc":{"op":"concat", "pad":"="}}}]}},
```

Comment:
If I'm reading this correctly the --output-fmt-option value appears to be the compression level